compiler/control/CompileMethod.cpp

```diff
@@ -312,6 +312,10 @@ compileMethodFromDetails(
             compiler.setDebug(createDebugObject(&compiler));
             }
 
+#ifdef VERIFY_IL
```

Comment:
Is there a reason why this is guarded with an `#ifdef` rather than a compiler option?  I'd rather have it live and enable-able via an option.